specs/_features/maxeb/beacon-chain.md

```diff
@@ -0,0 +1,405 @@
+    - [new `get_validator_excess_balance`](#new-get_validator_excess_balance)
+    - [updated  `is_partially_withdrawable_validator`](#updated--is_partially_withdrawable_validator)
+  - [Beacon state accessors](#beacon-state-accessors)
+    - [updated  `get_validator_churn_limit`](#updated--get_validator_churn_limit)
+  - [Beacon state mutators](#beacon-state-mutators)
+    - [updated  `initiate_validator_exit`](#updated--initiate_validator_exit)
+- [Genesis](#genesis)
+    - [updated  `initialize_beacon_state_from_eth1`](#updated--initialize_beacon_state_from_eth1)
+- [Beacon chain state transition function](#beacon-chain-state-transition-function)
+  - [Epoch processing](#epoch-processing)
+    - [updated  `process_registry_updates`](#updated--process_registry_updates)
+  - [Block processing](#block-processing)
+    - [updated  `get_expected_withdrawals`](#updated--get_expected_withdrawals)
+    - [Operations](#operations)
+      - [Deposits](#deposits)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+See [a modest proposal](https://notes.ethereum.org/@mikeneuder/increase-maxeb), the [diff view](https://github.com/michaelneuder/consensus-specs/pull/3/files) and 
+[security considerations](https://notes.ethereum.org/@fradamt/meb-increase-security).
+
+*Note:* This specification is built upon [Deneb](../../deneb/beacon-chain.md).
+
+## Constants
+
+The following values are (non-configurable) constants used throughout the specification.
+
+### Withdrawal prefixes
+
+| Name | Value |
+| - | - |
+| `BLS_WITHDRAWAL_PREFIX` | `Bytes1('0x00')` |
+| `ETH1_ADDRESS_WITHDRAWAL_PREFIX` | `Bytes1('0x01')` |
+| `COMPOUNDING_WITHDRAWAL_PREFIX` | `Bytes1('0x02')` |
+
+### Gwei values
+
+| Name | Value |
+| - | - |
+| `MIN_ACTIVATION_BALANCE` | `Gwei(2**5 * 10**9)`  (32 ETH) |
+| `MAX_EFFECTIVE_BALANCE_MAXEB` | `Gwei(2**11 * 10**9)` (2048 ETH) |
+
+## Containers
+
+### New containers
+
+### Extended Containers
+
+#### `BeaconState`
+
+*Note*: adding `activation_validator_balance` and `exit_queue_churn` fields to
+aid in rate limiting of activation and exit queue based on ETH rather than
+validator counts.
+
+```python
+class BeaconState(Container):
+    # Versioning
+    genesis_time: uint64
+    genesis_validators_root: Root
+    slot: Slot
+    fork: Fork
+    # History
+    latest_block_header: BeaconBlockHeader
+    block_roots: Vector[Root, SLOTS_PER_HISTORICAL_ROOT]
+    state_roots: Vector[Root, SLOTS_PER_HISTORICAL_ROOT]
+    historical_roots: List[Root, HISTORICAL_ROOTS_LIMIT]
+    # Eth1
+    eth1_data: Eth1Data
+    eth1_data_votes: List[Eth1Data, EPOCHS_PER_ETH1_VOTING_PERIOD * SLOTS_PER_EPOCH]
+    eth1_deposit_index: uint64
+    # Registry
+    validators: List[Validator, VALIDATOR_REGISTRY_LIMIT]
+    balances: List[Gwei, VALIDATOR_REGISTRY_LIMIT]
+
+    # --- NEW --- #
+    activation_validator_balance: Gwei
+    exit_queue_churn: Gwei 
+    # --- END NEW --- #
```

Comment:
The new fields have to be added at the end in`BeaconState`. e.g., after `historical_summaries`.

And `historical_summaries` is missing.

Reply:
adding! thanks